lib/evmone/instructions.hpp

```diff
@@ -1119,6 +1121,8 @@ inline DynStackResult printf(
     auto stack_pop = &stack_origin.top() - &handler.stack.top();
     assert(stack_pop < std::numeric_limits<int16_t>::max());
 
+    state.dyn_stack_pop = static_cast<unsigned>(stack_pop);
```

Comment:
Nit: it looks a bit hacky to only set `dyn_stack_pop` in printf, given that the function returns `stack_pop`. Is it possible to set `dyn_stack_pop` somewhere on the outer scope?

Reply:
You're right, we can assign it in the common handler. Fixed, thanks.